api/controllers/topics-controllers.js

```diff
@@ -1,8 +1,14 @@
 const {selectTopics} = require("../models/topics-models")
+const allEndPoints = require("../../endpoints.json")
 exports.getTopics = (req,res,next) => {
     selectTopics()
     .then((topics) => {
         res.status(200).send({topics})
     })
     .catch(next)
+}
+
+exports.getEndpoints = (req,res,next) => {
+    res.status(200).send(allEndPoints)
```

Comment:
This should be sent through on a key of an object like you ave done for topics - all endpoints is an object, but should be on a key of endpoints as well